### PR TITLE
Accept Pirep button not changing row inline #598

### LINKED ISSUE #598: Accept Pirep button not changing row inline
Not changing from "pending" to "accepted" unless refreshed

### PR DESCRIPTION
**Pull request by YashGovekar**

### CODE REVIEW

File: resources/views/admin/pireps/scripts.blade.php [hunk 1]
```diff
@@ -65,6 +65,23 @@
         changeStatus(values, (data) => {
           const destContainer = '#pirep_' + values.pirep_id + '_actionbar';
           $(destContainer).html(data);
+
+          const statusContainer = '#pirep_' + values.pirep_id + '_status_container';
+          let new_badge;
+          let new_badge_text;
+
+          if (values.new_status === '2') {
```
Comment (nabeelio) [inline resources/views/admin/pireps/scripts.blade.php:73]: Change this status to use the ENUM

**Comment by nabeelio** [comment]
Can you rename these PRs so they somewhat match the bug they're addressing? Thanks

File: resources/views/admin/pireps/scripts.blade.php [hunk 2]
```diff
@@ -65,6 +65,23 @@
...
+            new_badge = 'badge badge-success';
+            new_badge_text = 'Accepted'
+          }
+
+          if (values.new_status === '6') {
```
Comment (nabeelio) [inline resources/views/admin/pireps/scripts.blade.php:78]: Same as above

File: resources/views/admin/pireps/scripts.blade.php (hunk 1 above)
Comment (YashGovekar) [reply resources/views/admin/pireps/scripts.blade.php:73]: Alright

**Comment by YashGovekar** [comment]
> Can you rename these PRs so they somewhat match the bug they're addressing? Thanks

Okay sure.

File: resources/views/admin/pireps/scripts.blade.php [hunk 3]
```diff
@@ -70,12 +70,12 @@
           let new_badge;
           let new_badge_text;
 
-          if (values.new_status === '2') {
+          if (values.new_status === '{{ App\Models\Enums\PirepState::ACCEPTED }}') {
```
Comment (nabeelio) [inline resources/views/admin/pireps/scripts.blade.php:73]: You don't need the quoting, it should just be a plain int value

Comment (YashGovekar) [reply resources/views/admin/pireps/scripts.blade.php:73]: Yeah tried it but didn't work..

**Pull request merged**